packages/nitro-protocol/src/index.ts

```diff
@@ -53,7 +53,7 @@ export {
   hashOutcome,
 } from './contract/outcome';
 
-export {State, VariablePart, getVariablePart} from './contract/state';
+export {State, VariablePart, getVariablePart, getFixedPart} from './contract/state';
```

Comment:
?

Reply:
It's a function that  consumers of this package will probably want to use?

Reply:
Oh I misread this as an import, nevermind.